File: testprojects/csharp/benchmark/presentation/gui/facebook/LinkGUI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.facebook;

namespace CSharpBenchmark.presentation.gui.facebook
{

    //Functional requirement 3.1.2
    //Test case 72: Only class presentation.gui.facebook.LinkGUI may have a dependency with domain.facebook.Link
    //Result: FALSE
    public class LinkGUI
    {
        public LinkGUI()
        {
            //FR5.2
            Console.WriteLine(Link.linkUrl);
        }
    }
}

File: testresources/java/benchmark_2012/presentation/gui/facebook/LinkGUI.java
package presentation.gui.facebook;

import domain.facebook.Link;

//Functional requirement 3.1.2
//Test case 72: Only class presentation.gui.facebook.LinkGUI may have a dependency with domain.facebook.Link
//Result: FALSE
public class LinkGUI {
	public LinkGUI(){
		//FR5.2
		System.out.println(Link.linkUrl);
	}
}